src/main.rs

```diff
@@ -22,11 +22,11 @@ struct Cli {
     #[structopt(short = "t", long = "target")]
     target_result: Option<f64>,
 
-    #[structopt(long = "show-regression-line")]
-    show_regression_line: Option<bool>,
+    #[structopt(long = "show-regression-line", parse(try_from_str), default_value = "true")]
+    show_regression_line: bool,
 
     #[structopt(long = "show-target-line")]
-    show_target_line: Option<bool>,
+    show_target_line: bool,
```

Comment:
The idea is that both field should be specified only by using `--flag=true/false` so that you you can still change the default display without breaking previous usage that would use `--flag`.

I think the solution you used for `show_regression_line` is a bit nicer than the `Option` so you could just to that for this argument as well.
`#[structopt(long = "show-regression-line", parse(try_from_str), default_value = "false")]`

Reply:
If you add `default_value = "false"` you will get warned because it's the default behavior with boolean fields.

Reply:
Yes I know, that's why I used the Option 😉

Reply:
I don't understand why this matters? By default any boolean field will have the value `false` assigned by default, why using an optional change anything?

Reply:
That's a user side concern. I don't want the user to use `--flag` for certain options without explicitly writing `--flag=true/false` as the default behavior might evolve in a later version. I don't want to commit to the meaning of the implicit "`--flag`" as being true.